### PR TITLE
feat: add NGINX Fluentd example

### PR DESCRIPTION
**Pull request by f5-davey**

### CODE REVIEW

File: examples/fluentd/nginx/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        '"upstream_header_time":$upstream_header_time'
+    '}';
+
+access_log /var/log/nginx/apm.log apm;
+```
+
+Once enabled, JSON should be visible in the log for processed requests:
+
+```
+# tail /var/log/nginx/apm.log
```
Comment (F5-cheng) [inline examples/fluentd/nginx/README.md:37]: nit: maybe pipe to jq for easy reading.    `tail /var/log/nginx/apm.log | jq`

**Comment by F5-cheng** [comment]
Ship it!

File: examples/fluentd/nginx/README.md (hunk 1 above)
Comment (f5-davey) [reply examples/fluentd/nginx/README.md:37]: Agree `jq` makes it more readable.  In many cases though, I think jq is unlikely to be installed.  For that reason, I'm hesitant to add it.

File: examples/fluentd/README.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+The measurement is determined either via the via the `measurement` parameter within the Fluentd configuration for the Beacon plugin, or via the Fluentd tag.  The match directive for the Beacon plugin defines what Fluentd tags are routed to the Beacon output plugin.  Data within Fluentd events are mapped to tags and fields in a similar fashion.  If the `auto_tags` parameter is used for the Beacon plugin, then string values are mapped to tags and all else are mapped to fields.  The alternative is to use the `tag_keys` parameter to define the data elements that are mapped to tags.
```
Comment (focrensh) [inline examples/fluentd/README.md:5]: small repeat typo

Comment (f5-davey) [reply examples/fluentd/README.md]: Fixed.  Thanks!

**Pull request merged**